files/en-us/web/api/xranchor/index.html

```diff
@@ -0,0 +1,71 @@
+---
+title: XRAnchor
+slug: Web/API/XRAnchor
+tags:
+  - API
+  - Interface
+  - Reference
+  - WebXR
+  - XR
+  - AR
+  - VR
+browser-compat: api.XRAnchor
+---
+<p>{{APIRef("WebXR Device API")}} {{secureContext_header}}</p>
+
+<p>The <code><strong>XRAnchor</strong></code> interface creates anchors which keep track of the pose that is fixed relative to the real world. With anchors, you can specify poses in the world that need to be updated to correctly reflect the evolving understanding of the world, such that the poses remain aligned with the same place in the physical world. The helps to build an illusion that the placed objects are really present in the user’s environment.</p>
```

Comment:
```suggestion
<p>The <code><strong>XRAnchor</strong></code> interface creates anchors which keep track of the pose that is fixed relative to the real world. With anchors, you can specify poses in the world that need to be updated to correctly reflect the evolving understanding of the world, such that the poses remain aligned with the same place in the physical world. That helps to build an illusion that the placed objects are really present in the user’s environment.</p>
```
(or *“This”* if you prefer…)